DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -936,9 +952,9 @@ public async Task<IReadOnlyList<DiscordRole>> ModifyRolePositionsAsync(IDictiona
     /// <exception cref="NotFoundException">Thrown when the member does not exist.</exception>
     /// <exception cref="BadRequestException">Thrown when an invalid parameter was provided.</exception>
     /// <exception cref="ServerErrorException">Thrown when Discord is unable to process the request.</exception>
-    public async Task BanMemberAsync(DiscordMember member, int delete_message_days = 0, string reason = null)
+    public async Task BanMemberAsync(DiscordMember member, int delete_message_days = 0, string? reason = null)
```

Comment:
here also, ups my bad

Reply:
see the review note i left - please don't change this in this PR (any further)

Reply:
(too late, sorry)